src/main/java/seedu/address/commons/util/JsonUtil.java

```diff
@@ -49,14 +49,16 @@ static <T> T deserializeObjectFromJsonFile(Path jsonFile, Class<T> classOfObject
     }
 
     /**
-     * Returns the Json object from the given file or {@code Optional.empty()} object if the file is not found.
-     * If any values are missing from the file, default values will be used, as long as the file is a valid json file.
+     * Returns the Json object from the given file or {@code Optional.empty()} object if the file is
+     * not found. If any values are missing from the file, default values will be used, as long as
+     * the file is a valid json file.
+     *
      * @param filePath cannot be null.
      * @param classOfObjectToDeserialize Json file has to correspond to the structure in the class given here.
-     * @throws DataConversionException if the file format is not as expected.
+     * @throws DataConversionException if the loading of file failed
```

Comment:
Not sure if `loading of the file` will be understood as both 'reading the file content and parsing the content'. Perhaps use `loading of JSON data from the file`?

Reply:
For this version of PR, the handling of trying to read file content and parsing the content is both handled within `JsonUtil` so in a sense the new renamed exception represents both 'reading the file content and parsing the content'.